kotlin-reference-server/src/main/kotlin/org/stellar/reference/event/processor/Sep31EventProcessor.kt

```diff
@@ -0,0 +1,121 @@
+package org.stellar.reference.event.processor
+
+import kotlinx.coroutines.runBlocking
+import org.stellar.anchor.api.callback.GetCustomerRequest
+import org.stellar.anchor.api.platform.*
+import org.stellar.anchor.api.rpc.method.RpcMethod
+import org.stellar.anchor.api.sep.SepTransactionStatus.*
+import org.stellar.reference.client.PlatformClient
+import org.stellar.reference.data.*
+import org.stellar.reference.di.ServiceContainer.customerService
+import org.stellar.reference.di.ServiceContainer.sepHelper
+import org.stellar.reference.log
+
+class Sep31EventProcessor(
+  private val platformClient: PlatformClient,
+) : SepAnchorEventProcessor {
+  companion object {
+    val requiredKyc =
+      listOf("bank_account_number", "bank_account_type", "bank_number", "bank_branch_number")
+  }
+
+  override suspend fun onQuoteCreated(event: SendEventRequest) {
+    TODO("Not yet implemented")
+  }
+
+  override suspend fun onTransactionCreated(event: SendEventRequest) {
+    log.warn { "Sep31EventProcessor onTransactionCreated" }
```

Comment:
Nit: I don't think we need the Sep31EventProcessor` prefix as the logger should handle this.

Reply:
Will remove this ... it was for print log purpose